vmm/src/vm.rs

```diff
@@ -425,6 +461,12 @@ impl Vm {
             &cmdline_cstring,
         )
         .map_err(Error::LoadCmdLine)?;
+
+        let initramfs_config = match self.initramfs {
+            Some(_) => Some(self.load_initramfs(mem.deref())?),
+            None => None,
+        };
+
         let boot_vcpus = self.cpu_manager.lock().unwrap().boot_vcpus();
         let _max_vcpus = self.cpu_manager.lock().unwrap().max_vcpus();
 
```

Comment:
You need to switch back to `BootProtocol::LinuxBoot` if initramfs is specified as PVH doesn't support initrd. Print a warning with `warn!`

Reply:
sorry, I don't follow.